app/jobs/autotest_specs_job.rb

```diff
@@ -12,7 +12,9 @@ def perform(host_with_port, assignment_id)
     server_path = MarkusConfigurator.autotest_server_dir
     server_username = MarkusConfigurator.autotest_server_username
     server_command = MarkusConfigurator.autotest_server_command
-    server_params = { markus_address: markus_address, assignment_id: assignment_id }
+    test_specs_path = File.join(assignment_tests_path, assignment.get_test_specs_name)
+    test_specs = JSON.load File.read(test_specs_path)
+    server_params = { markus_address: markus_address, assignment_id: assignment_id, test_specs: test_specs }
```

Comment:
The `specs.json` file will not be copied over in the end. We're just doing it now because we don't have an interface to upload it otherwise.  This is just a hack to get things working for now.